vizro-core/examples/scratch_dev/app.py

```diff
@@ -1,36 +1,28 @@
 """Dev app to try things out."""
 
-import pandas as pd
 import vizro.models as vm
-import vizro.plotly.express as px
 from vizro import Vizro
-from vizro.models.types import capture
-
-animals = pd.DataFrame(
-    {"animals": ["giraffes", "orangutans", "monkeys"], "value": [20, 14, 23], "color": ["Male", "Female", "Female"]}
-)
+import vizro.plotly.express as px
+import plotly.io as pio
 
 iris = px.data.iris()
 
-
-@capture("graph")
-def bar(data_frame):
-    """LA LA LA LA."""
-    fig = px.bar(data_frame, x="sepal_length", y="sepal_width", color="species")
-    return fig
-
-
-@capture("graph")
-def px_bar(data_frame):
-    """LA LA LA LA."""
-    fig = px.bar(data_frame, x="animals", y="value", color="color")
-    return fig
+# TODO: Ask A. why this does not work?
+pio.templates["vizro_dark"]["layout"]["font_family"] = "PlayfairDisplay, Inter, sans-serif, Arial, serif"
+pio.templates["vizro_light"]["layout"]["font_family"] = "PlayfairDisplay, Inter, sans-serif, Arial, serif"
```

Comment:
@antonymilne - I thought this would work, but it doesn't. Any ideas? If we want to change the font of the chart, do we have to do it via post update calls for each chart?

Reply:
I think I found out why.. I'll create a separate PR to fix the bug

Reply:
https://github.com/mckinsey/vizro/pull/718